api/v1alpha1/ramenconfig_types.go

```diff
@@ -155,6 +155,9 @@ type RamenConfig struct {
 
 	// Unprotect deleted or deselected PVCs
 	VolumeUnprotectionEnabled bool `json:"volumeUnprotectionEnabled,omitempty"`
+
+	// RamenOpsNamespace is the namespace where resources for unmanaged apps are created
+	RamenOpsNamespace string `json:"ramenOpsNamespace,omitempty"`
```

Comment:
Gowtham pointed out that we should prevent change of this value once it is set.

Reply:
It is somewhat restrictive, but I understand why Gowtham is concerned about changing it once it is been set. However, if the user were to delete all the unmanaged apps, wouldn't that provide an opportunity for the user to make changes?

Reply:
Also, this would be an admin change, i.e the CM would be editable for an admin persona, and if they change it from a->b, DRPCs in a will report errors. I am sort of fine with that, cleanup `a` and move to NS `b`.

The alternatives would put some stress on the code which is better avoided.

But good observation though.